documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,38 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application and reflects the new changes.
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+![Oauth client creation](../images/oauthClientCreation.png)
+
+Select the client tab, which provides an option to register a new client as shown below:
+
+![Register new client](../images/registerNewOauthClient.png)
+
+During registering the client, in the subscription tab, select the portal landscape you would like to subscribe to, ex: `portal/nwc` as shown below:
+
+![Portal subscription](../images/portalSubscription.png)
+
+After saving these changes, a client secret is generated. Ensure to make note of the newly generated secret as it is not available later.
```

Comment:
Generating a client secret is wrong here. The user needs to change Authorization Grant to 'Client Credentials' and type a password themselves, only the ID is generated. An image showing this would be helpful.

Reply:
Good point 👍